### ISSUE TITLE
V1.7.1 Can't go after install step 2

### ISSUE THREAD

**Issue by adamhesim** [HotaruCMS/HotaruCMS#98]
Hi,

I can't finish the install of the 1.7.1. I try to install Hotaru in his own Apache Vhost. The mysql server is runnig, user and password are OK but i keep getting these errors at step 3 : 

```
Notice: Use of undefined constant TABLE_USERS - assumed 'TABLE_USERS' in /HotaruCMS/install/index.php on line 440

Fatal error: Call to a member function get_var() on null in /HotaruCMS/install/index.php on line 441
```

If I var_dump($h) in the step 3 select, /install/index.php , I can see that $h->db is null : 
```PHP
	case 3:  // /install/index.php: LINE 199
                // Remove any cookies set in a previous installation:
                setcookie("hotaru_user", "", time()-3600, "/");
                setcookie("hotaru_key", "", time()-3600, "/");
                        
                // remove cookies from whole domain just in case of 1.4.2 cookies issue
                $parsed = parse_url(BASEURL); 
                setcookie("hotaru_user", "", time()-3600, "/", "." . $parsed['host']);
                setcookie("hotaru_key", "", time()-3600, "/", "." . $parsed['host']);
                
		if ($action == 'upgrade') {
			upgrade_step_3($h);
		} else {
			$db = init_database();              
            var_dump($h);
			register_admin($h);           // Username and password for Admin user...
		}
		break;
```
I assume that is why I get the fatal error.
And I can be sure that it's not a DB login issue because at step 2 the tables where actually created, with the settings.php infos.

**Comment by adamhesim** [comment]
Looks like some user on the forum get the same errors and solved it but i can't see how and despite my inscription i can't post on the forum : 
http://forums.hotarucms.org/threads/download-hotaru-cms-core.6964/page-14#post-22124

**Comment by shibuya246** [comment]
I think this was solved in 1.7.2
